Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,11 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.app {
+  height: 500px;
+  width: 45%;
+  margin: 1em auto;
```

Comment:
Use `rem` or `px` units. `em` units depend on the font-size of this particular component, which is unset and therefore would revert to browser font, making it unpredictable.